File: samples/php/extract-tags.php
<?php

$curl = curl_init();

$apiKey = "";    // Replace with your API Key
$trackId = "";  // Replace with Id of the track

$includeTags = array("CONTENT TYPE", "GENRE V2", "MOOD", "BPM", "KEY SHARP", "ENERGY", "INSTRUMENTATION", "HIT POTENTIAL");
$requestBody = json_encode( array( "id"=>$trackId , "tags"=> $includeTags ) );

$authHeader = $apiKey.":"."";
$encodedAuthHeader = base64_encode($authHeader);

curl_setopt_array($curl, array(
  CURLOPT_URL => "https://api-us.musiio.com/api/v1/extract/tags",
  CURLOPT_RETURNTRANSFER => true,
  CURLOPT_ENCODING => "",
  CURLOPT_MAXREDIRS => 10,
  CURLOPT_TIMEOUT => 0,
  CURLOPT_FOLLOWLOCATION => true,
  CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
  CURLOPT_CUSTOMREQUEST => "POST",
  CURLOPT_POSTFIELDS => $requestBody,
  CURLOPT_HTTPHEADER => array(
    "Authorization: Basic ".$encodedAuthHeader,
	"Content-Type: application/json"
  ),
));

$response = curl_exec($curl);
curl_close($curl);
echo $response;
?>

File: samples/tagging/python-individual/extract-tags.py
import requests
from requests.auth import HTTPBasicAuth
import json

api_key = ""    # Your API Key
track_id = ""   # Track ID

url = "https://api-us.musiio.com/api/v1/extract/tags"

# You can customize tagging results by adding or removing query element from "tags" below
payload = {
    "id": track_id,
    "tags": ["CONTENT TYPE", "GENRE V3", "MOOD", "BPM", "KEY", "KEY SHARP", "ENERGY", "INSTRUMENTATION"]
}

headers = {
  'Content-Type': 'application/json'
}

response = requests.post(url, auth=HTTPBasicAuth(api_key, ''), headers=headers, data=json.dumps(payload))

print(response.text.encode('utf8'))
